.vale/fixtures/RedHat/TermsErrors/testvalid.adoc

```diff
@@ -110,6 +112,8 @@ drag
 dual-boot
 dynamically
 earlier
+ELB
```

Comment:
Remove ELB from here. ELB is not in the `TermsErrors.yml` rule. 

```suggestion
```

Reply:
Sorry about this back-and-forth. Updated.